packages/desktop/components/popups/send/SendConfirmationPopup.svelte

```diff
@@ -203,7 +202,7 @@
     >
     <div class="w-full flex-col space-y-2">
         {#if transactionDetails.type === NewTransactionType.TokenTransfer}
-            <BasicActivityDetails {activity} {unit} />
+            <BasicActivityDetails {activity} />
```

Comment:
If you remove the unit prop this breaks the current send flow, because if a user selects glow as a unit, it won't show up in the confirmation popup anymore. It should be fine as it is, but let's keep this PR unmerged until we have a release branch. Additionally, also remove the unused prop in `BasicActivityDetails`.

Reply:
the unit is already in `transactionDetails` and therefore also in `activity`